### PR TITLE
[Release_3.0]modify elementpath version ==2.5.0

### PR DESCRIPTION
**Pull request by wenlingz**
modify elementpath version ==2.5.0

Tracked-On:projectacrn#6688
Signed-off-by: wenlingz <wenling.zhang@intel.com>

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

**Comment by gvancuts** [comment]
Can you fix your commit message to have a subject line and a message body?

**Review by dbkinder — CHANGES_REQUESTED** [review]

File: debian/control [hunk 1]
```diff
@@ -34,7 +34,7 @@ Build-Depends:
    python3-lxml,
    python3-defusedxml,
    python3-xmlschema (>= 1.10.0),
-   python3-elementpath (>= 2.5.0),
+   python3-elementpath (== 2.5.0),
```
[the remark this answers is not in the document]
Comment (wenlingz) [reply debian/control]: Modified, please check, thanks

**Review by dbkinder — APPROVED** [review]
LTGM, thanks.

**Comment by szhen11** [comment]
@yakuizhao Please help to approve this PR.

**Review by yakuizhao — APPROVED** [review]
LGTM

**Pull request merged**